conda-recipes/qsharp/meta.yaml

```diff
@@ -13,28 +13,19 @@ requirements:
     - python
     - setuptools
     - jupyter
-    - jupyter_client<7.0
     - iqsharp={{ version }}
 
   run:
     - python
-    - pyzmq
     - iqsharp={{ version }}
 
-  run_constrained:
-    # We forbid SciPy version 1.8.1 due to incompatibilities with some versions
-    # of PyZMQ and GCC, leading to runtime dynamic link failures of the form
-    # "version `GLIBCXX_3.4.30' not found (required by /miniconda/lib/python3.9/site-packages/scipy/fft/_pocketfft/pypocketfft.cpython-39-x86_64-linux-gnu.so)"
-    - scipy!=1.8.1 # [linux]
```

Comment:
I have run into the SciPy 1.8.1 bug even on versions other than 3.7; it's a problem with the DLL linking on their implementation of the FFT, such that we may want to keep the narrow carve out here.

Reply:
I don't see the problem happening anymore, so I'm going to keep it out.

Reply:
@cgranade @anpaz With buster going eol next month, might be worth moving the base image to `python:3.9-slim-bullseye`

Reply:
Fully agreed, @idavis; would be good to have this in for the next release as a result of that.